include/zephyr/bluetooth/uuid.h

```diff
@@ -1212,139 +2103,535 @@ struct bt_uuid_128 {
+/**
+ *  @brief GATT Characteristic Two Zone Heart Rate Limits UUID Value
+ */
+#define BT_UUID_GATT_2ZHRL_VAL 0x2a95
+/**
+ *  @brief GATT Characteristic Two Zone Heart Rate Limits
+ */
+#define BT_UUID_GATT_2ZHRL \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_2ZHRL_VAL)
+/**
+ *  @brief GATT Characteristic VO2 Max UUID Value
+ */
+#define BT_UUID_GATT_VO2_MAX_VAL 0x2a96
+/**
+ *  @brief GATT Characteristic VO2 Max
+ */
+#define BT_UUID_GATT_VO2_MAX \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_VO2_MAX_VAL)
+/**
+ *  @brief GATT Characteristic Waist Circumference UUID Value
+ */
+#define BT_UUID_GATT_WC_VAL 0x2a97
+/**
+ *  @brief GATT Characteristic Waist Circumference
+ */
+#define BT_UUID_GATT_WC \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_WC_VAL)
+/**
+ *  @brief GATT Characteristic Weight UUID Value
+ */
+#define BT_UUID_GATT_WEIGHT_VAL 0x2a98
+/**
+ *  @brief GATT Characteristic Weight
+ */
+#define BT_UUID_GATT_WEIGHT \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_WEIGHT_VAL)
+/**
+ *  @brief GATT Characteristic Database Change Increment UUID Value
+ */
+#define BT_UUID_GATT_DBCHINC_VAL 0x2a99
+/**
+ *  @brief GATT Characteristic Database Change Increment
+ */
+#define BT_UUID_GATT_DBCHINC \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_DBCHINC_VAL)
+/**
+ *  @brief GATT Characteristic User Index UUID Value
+ */
+#define BT_UUID_GATT_USRIDX_VAL 0x2a9a
+/**
+ *  @brief GATT Characteristic User Index
+ */
+#define BT_UUID_GATT_USRIDX \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_USRIDX_VAL)
+/**
+ *  @brief GATT Characteristic Body Composition Feature UUID Value
+ */
+#define BT_UUID_GATT_BCF_VAL 0x2a9b
+/**
+ *  @brief GATT Characteristic Body Composition Feature
+ */
+#define BT_UUID_GATT_BCF \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_BCF_VAL)
+/**
+ *  @brief GATT Characteristic Body Composition Measurement UUID Value
+ */
+#define BT_UUID_GATT_BCM_VAL 0x2a9c
+/**
+ *  @brief GATT Characteristic Body Composition Measurement
+ */
+#define BT_UUID_GATT_BCM \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_BCM_VAL)
+/**
+ *  @brief GATT Characteristic Weight Measurement UUID Value
+ */
+#define BT_UUID_GATT_WM_VAL 0x2a9d
+/**
+ *  @brief GATT Characteristic Weight Measurement
+ */
+#define BT_UUID_GATT_WM \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_WM_VAL)
+/**
+ *  @brief GATT Characteristic Weight Scale Feature UUID Value
+ */
+#define BT_UUID_GATT_WSF_VAL 0x2a9e
+/**
+ *  @brief GATT Characteristic Weight Scale Feature
+ */
+#define BT_UUID_GATT_WSF \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_WSF_VAL)
+/**
+ *  @brief GATT Characteristic User Control Point UUID Value
+ */
+#define BT_UUID_GATT_USRCP_VAL 0x2a9f
+/**
+ *  @brief GATT Characteristic User Control Point
+ */
+#define BT_UUID_GATT_USRCP \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_USRCP_VAL)
+/**
+ *  @brief Magnetic Flux Density - 2D Characteristic UUID value
+ */
+#define BT_UUID_MAGN_FLUX_DENSITY_2D_VAL 0x2aa0
+/**
+ *  @brief Magnetic Flux Density - 2D Characteristic
+ */
+#define BT_UUID_MAGN_FLUX_DENSITY_2D \
+	BT_UUID_DECLARE_16(BT_UUID_MAGN_FLUX_DENSITY_2D_VAL)
```

Comment:
I'm having a hard time figuring out when the UUIDs are infixed with GATT and when they are not. It seems that that the majority of the new UUIDs are infixed with GATT, but all the existing GATT characteristics are still without GATT (which makes sense). 

Have you considered not infixing any with "GATT" to be more consistent?

Reply:
As far as I understood the specification, all UUIDs in range `0x2a00-0x2bf5` belong to GATT Characteristics. From this perspective more consistent would be exactly opposite - infixing all of them with "GATT". But I'm not sure.

Reply:
The specification never specifies GATT for the characteristics, although I agree that the characteristic UUIDs are probably only used for GATT, similar to the Declaration and Descriptors which are indeed infixed with `_GATT_`. 

My only argument is that the majority of existing characteristics are not infixed with GATT, but perhaps it's better to add the new ones with the GATT infix, and then, in a new PR, add the infix to the existing ones. 

No objections to add them with the infix